man/systemd.swap.xml

```diff
@@ -94,10 +94,10 @@
     dependencies on the device units or the mount units of the files
     they are activated from.</para>
 
-    <para>Swap units with <varname>DefaultDependencies=</varname> in the <literal>[Unit]</literal> section enabled
-    implicitly acquire a <varname>Conflicts=</varname> and an <varname>After=</varname> dependency on
-    <filename>umount.target</filename> so that they are deactivated at shutdown, unless
-    <varname>DefaultDependencies=no</varname> is specified.</para>
+    <para>Swap units with <varname>DefaultDependencies=</varname> set to its default <option>yes</option> value in the
+    <literal>[Unit]</literal> section enabled implicitly acquire a <varname>Conflicts=</varname> and a
+    <varname>Before=</varname> dependency on <filename>umount.target</filename> so that they are deactivated at
+    shutdown as well as a <varname>Before=swap.target</varname> dependency.
```

Comment:
Erk, sorry. I took a second or two (but not much time) to look for instructions on how to build the docs; clearly should have taken longer. Good thing the CI catches it!  (As did my editor and I ignored it, apparently.) Fixed.